Classes/RedirectService.php

```diff
@@ -65,6 +65,9 @@ public function buildResponseIfApplicable(Request $httpRequest)
             }
             return $this->buildResponse($httpRequest, $redirect);
         } catch (\Exception $exception) {
+            if ($exception instanceof Exception) {
```

Comment:
Maybe add a comment explaining the difference between `\Exception` and `Exception` at that point and quickly clear up the intention of that clause? Otherwise someone might try to "clean it up" later… :)